### PR TITLE
First translation from French to English

### PR DESCRIPTION
**Pull request by mlep**

### CODE REVIEW

File: en/User/Framabag_account_creation.md [hunk 1]
```diff
@@ -6,4 +6,30 @@ subTitle: Framabag account creation
...
+## Account creation
+
+Click on the button **Créez votre compte / Create your account**.
+
+You have to input some information: your username, your password and your email address. Your address is used only to validate your account, except if you check the box to subscribe the newsletter (3 to 4 messages per year).
+
+Once this form validated, you will receive an email with the confirmation link. Click on it to create your account.
+
+You will receive a last email with the address of your Framabag account, which will look like
+`https://www.framabag.org/u/votre-nom-d'utilisateur`.
```
Comment (nicosomb) [inline en/User/Framabag_account_creation.md]: You can maybe replace `votre-nom-d'utilisateur` with `your-username` ;-)

Comment (mlep) [inline en/User/Framabag_account_creation.md]: Oopss...
However, I cannot find how to do it in GitHub.

Comment (tcitworld) [inline en/User/Framabag_account_creation.md]: Go to the file in the Github interface, then click on edit on top right of the text area.

Comment (tcitworld) [inline en/User/Framabag_account_creation.md]: (the pencil)

Comment (mlep) [inline en/User/Framabag_account_creation.md]: Found! Thank you.
Done.

**Pull request merged**